aiohttp_jinja2/__init__.py

```diff
@@ -49,6 +51,10 @@ def render_string(template_name, request, context, *, app_key):
         text = "context should be mapping, not {}".format(type(context))
         # same reason as above
         raise web.HTTPInternalServerError(reason=text, text=text)
+    if REQUEST_CONTEXT_KEY in request:
+        for k, v in request.get(REQUEST_CONTEXT_KEY, {}).items():
+            if k not in context:
```

Comment:
I think we should be very explicit here by raising exception if context keys overlap each other.